lib/__tests__/_sendEvent.test.ts

```diff
@@ -54,6 +55,20 @@ describe("sendEvent", () => {
         ]
       });
     });
+    it("should include X-Algolia-* query parameters", () => {
+      (AlgoliaInsights as any).sendEvent("click", {
+        eventName: "my-event",
+        index: "my-index",
+        objectIDs: ["1"]
+      });
+      const [, requestUrl] = XMLHttpRequest.open.mock.calls[0];
```

Comment:
```suggestion
      const requestUrl = XMLHttpRequest.open.mock.calls[0][1];
```

I don't see much advantage of the destructuring here TBH, but no big deal either way

Reply:
remove there and in another place: 3cf7fd2d7da925b2b04da02acc324e80566360f0